packages/expo-cli/src/urlOpts.ts

```diff
@@ -64,8 +67,12 @@ async function optsAsync(projectDir: string, options: any) {
   }
 
   let rawArgs = options.parent.rawArgs;
+
   if (hasBooleanArg(rawArgs, 'dev')) {
     opts.dev = getBooleanArg(rawArgs, 'dev');
+  } else {
+    opts.dev = true;
+    opts.minify = false;
```

Comment:
```suggestion
```
`opts.minify = false;` should be in the other `if-else` below that checks for the `--minify` flag.